obspy/io/sac/sactrace.py

```diff
@@ -764,13 +764,9 @@ def __init__(self, leven=True, delta=1.0, b=0.0, e=0.0, iztype='ib',
                   'lcalda': lcalda, 'lpspol': lpspol, 'lovrok': lovrok,
                   'internal0': internal0}
 
-        # required = ['delta', 'b', 'npts', ...]
-        # provided = locals()
-        # for hdr in required:
-        #     header[hdr] = kwargs.pop(hdr, provided[hdr])
-
         # combine header with remaining non-required args.
-        # XXX: user can put non-SAC key:value pairs into the header.
+        # user can put non-SAC key:value pairs into the header, but they're
+        # ignored on write.
```

Comment:
Any easy possibility to raise here? I think that for the `SACTrace` class it would be nice if it raises if an unknown header is set as that in all likeliness is an error.